src/core/listener.c

```diff
@@ -769,6 +769,17 @@ QuicListenerParamSet(
         return QUIC_STATUS_SUCCESS;
     }
 
+    if (Param == QUIC_PARAM_LISTENER_QTIP) {
```

Comment:
Probably convert this function back to using a switch statement now that there is more than one.